app/views/file_imports/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Edit <%= @file_import.active_storage_attachment.blob.filename %></h2>
+
+<%= form_with model: @file_import, class: "bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4" do |f| %>
+  <div class="mb-4">
+    <%= f.label :status, class: "form-label" %>
+    <%= f.select :status, options_for_select(FileImport.statuses.map { |k, v| [k.capitalize, k] }), class: "form-field" %>
```

Comment:
Let's use [humanize](https://api.rubyonrails.org/v6.1.4/classes/ActiveSupport/Inflector.html#method-i-humanize) in case we ever have a status with an underscore.